internal/olm/operator/registry/index_image.go

```diff
@@ -138,6 +139,13 @@ func (c *IndexImageCatalogCreator) BindFlags(fs *pflag.FlagSet) {
 		"while pulling bundles")
 	fs.BoolVar(&c.UseHTTP, "use-http", false, "use plain HTTP for container image registries "+
 		"while pulling bundles")
+	fs.StringVar(&c.ImagePullPolicy, "image-pull-policy", string(corev1.PullAlways), "image pull policy for the registry pod")
```

Comment:
only this flag would not solve the problem?

Reply:
It would but I would rather not type out the string every time or lookup possible values. Just personal preference.

Reply:
@mateusoliveira43 is correct, IMO this PR is encapsulating the functionally of the flag and not letting a user choose the value(s) they want. I think it's better to just go with one flag to expose a given pods `ImagePullPolicy`.

Reply:
Ok will update in a bit